config/application.rb

```diff
@@ -11,6 +11,10 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
     config.assets.initialize_on_precompile = false
+    Raven.configure do |config|
+      config.dsn = "https://44a0f5e4de3748e18055619564381711:36ad89df984b45eb83e877df1090cfdc@o384221.ingest.sentry.io/5215099"
```

Comment:
We should move it to ENV. 

We will have separate sentry projects for staging and production